### ISSUE TITLE
Reprocessing same snapshot over and over.

### ISSUE THREAD

**Issue by mkaltner** [djdd87/SynoAI#219]
Has anyone else seen this behavior?  Seems to be recent.  I'll get hundreds of telegram messages of the same image.

**Comment by djdd87** [comment]
Nope, never seen that. And the software hasn't changed for a year, so it's unlikely to be a fault with SynoAI.

**Comment by jib-au** [comment]
Yeah I have had this issue recently

**Comment by jib-au** [comment]
I just resolved this issue. For some reason my deepstack container crashed. It was spitting out a 503 error. I stopped the container, verified vision-detection set to true, started back up and resolved the issue. Its been stable for almost a year.

**Comment by jib-au** [comment]
I was getting this issue again, and I wrote a script to purge the folders in the SynoAI\Captures\ directory every 10 seconds and now I am getting updated snapshots again. @djdd87 appreciate you have been flat out, but is this something you could play with in your dev environment, perhaps have a mechanism to simply overwrite the snapshots in the captures folder so there is only the latest snapshot for that camera. I don't think its necessary holding onto those snapshots, as you should get the notification and time from your notifier. Thanks.

**Comment by djdd87** [comment]
I'll to confirm, but I thought the snapshots were saved with unique names. Or is that not the case/am I not understanding the issue?

(Also - I'm writing version 2 at the moment and it's going quite well.)

**Comment by jib-au** [comment]
Yeah I believe the snapshots are saved with unique names. I think this all depends on the notifier. I'm using Home Assistant (HA) as my notifier, perhaps the issue is not present with push bullet or telegram. It might even be an issue on Home Assistant. Since my captures folders in synoAI is being purged every 10 seconds, the webhook being sent to HA has been sending me the correct snapshot except when a person is detected in a single event and it sent 5 notifications , not all of them will be current as the 10 second buffer can have an old image in there and send the old image. But it is a lot better. I think having an option in the syno config to simply overwrite the image would improve this for HA notifiers (not sure about others). Note: HA stores the latest image from the webhook and I can view the images inside HA which is useful if I need to view the latest snapshot. 

Look forward to the new release!

**Comment by djdd87** [comment]
I'm not sure if this is related, but I've noticed my own snapshots now are basically always the same within any 2 second time period. So if I get a snapshot at 12:00:00 and 12:00:01, the images will be identical. However at 12:00:02 I get a new image. I think something has changed in SSS.
